extensions/raycast/json_to_csv.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
You don't need to use paramaters. This function should take what's in your clipboard, modify it and update your keyboard